docs/api.rst

```diff
@@ -69,6 +69,8 @@ lte           :code:`<=`                less than or equal
 neq           :code:`<>` or :code:`!=`  not equal
 like          :code:`LIKE`              LIKE operator (use * in place of %)
 ilike         :code:`ILIKE`             ILIKE operator (use * in place of %)
+match         :code:`~`                 ~ operator
+imatch        :code:`~*`                ~* operator
```

Comment:
A link to https://www.postgresql.org/docs/current/functions-matching.html#FUNCTIONS-POSIX-REGEXP would be nice here.

Reply:
Started a small section for pattern matching, with references to the Pattern Matching official docs as well as the index section in module pg_trgm.